### PR TITLE
fix: copy changes 221102

### LINKED ISSUE #1156: Copy changes suggested by IRIS
Please view IRIS copy suggestion for the hard launch in the attached document:

[5 Oct - Copy inserts.docx](https://github.com/accessibility-exchange/platform/files/9759343/5.Oct.-.Copy.inserts.docx)

### PR DESCRIPTION
**Pull request by chosww**
Make copy changes on #1156.

### Prerequisites

If this PR changes PHP code or dependencies:

- [x] I've run `composer format` and fixed any code formatting issues.
- [x] I've run `composer analyze` and addressed any static analysis issues.
- [x] I've run `php artisan test` and ensured that all tests pass.
- [x] I've run `composer localize` to update localization source files and committed any changes.

If this PR changes CSS or JavaScript code or dependencies:

- [x] I've run `npm run lint` and fixed any linting issues.
- [x] I've run `npm run build` and ensured that CSS and JavaScript assets can be compiled.

### CODE REVIEW

**Comment by jobara** [comment]
Based on the word doc from #1156 I'm not seeing the following changes in this PR.

- Sign up pathway:
  - 9. Cut: Individual Seeking Training 
  - Replace with: Training Participant
- Your details:
  - 11. Cut: "if you change this email..."
- FRO Dashboard:
  - 12. Insert: Learn  about engaging disability communities
- Projects
  - 13. Cut: Projects I'm running
  - Insert: My Projects

**Comment by chosww** [comment]
> Based on the word doc from #1156 I'm not seeing the following changes in this PR.
> 
> * Sign up pathway:
>   
>   * 9. Cut: Individual Seeking Training
>   * Replace with: Training Participant
> * Your details:
>   
>   * 11. Cut: "if you change this email..."
> * FRO Dashboard:
>   
>   * 12. Insert: Learn  about engaging disability communities
> * Projects
>   
>   * 13. Cut: Projects I'm running
>   * Insert: My Projects

Hi @jobara, I met @cherylhjli yesterday to talk about some copies and she told me not to make those updates. I will ask her to double check.

File: app/Enums/UserContext.php [hunk 1]
```diff
@@ -15,18 +15,18 @@ public static function labels(): array
         return [
             'individual' => __('Individual'),
             'organization' => __('Community Organization'),
-            'regulated-organization' => __('Regulated Organizations: Business, Federal Government and Public Sector Organizations'),
+            'regulated-organization' => __('Federally Regulated Organizations'),
```
Comment (greatislander) [inline app/Enums/UserContext.php]: I don't think this should be plural if the others aren't.

**Review by greatislander — CHANGES_REQUESTED** [review]

**Comment by cherylhjli** [comment]
> 
Yes, don't worry about those changes. However I just found out today we still have to do this one:

Sign up pathway:

        Cut: Individual Seeking Training
    Replace with: Training Participant


Thanks!

File: app/Enums/UserContext.php (hunk 1 above)
Comment (chosww) [reply app/Enums/UserContext.php]: The copy suggestions say it in plural, should we just make it singular?

Comment (greatislander) [reply app/Enums/UserContext.php]: Yeah I think the copy suggestion is wrong. Thoughts, @cherylhjli / @michelled?

Comment (greatislander) [reply app/Enums/UserContext.php]: The question is:

**Tell us who you are joining the Accessibility Exchange as**

The answers are:

- Individual
- Community Organization
- Federally Regulated Organization

If the last one is plural it doesn't fit

**Comment by chosww** [comment]
> > 
> 
> Yes, don't worry about those changes. However I just found out today we still have to do this one:
> 
> Sign up pathway:
> 
> ```
>     Cut: Individual Seeking Training
> Replace with: Training Participant
> ```
> 
> Thanks!

Seems like we don't have that option?

**Comment by greatislander** [comment]
> > Yes, don't worry about those changes. However I just found out today we still have to do this one:
> > Sign up pathway:
> > ```
> >     Cut: Individual Seeking Training
> > Replace with: Training Participant
> > ```

> Seems like we don't have that option?

Yeah, that's not available yet— it will be added when @chosww finishes the training modules. So it can be left out for now. Just make sure it's in the strings.php file: https://github.com/accessibility-exchange/platform/blob/ab7d6390538b23e0526178a27bdcbe1c98f0c6de/app/strings.php

**Comment by greatislander** [comment]
@chosww We'll need to add this to the UserContext enum:

```php
case TrainingParticipant = 'training-participant';
```

With the appropriate label (`__('Training Participant')`) and description from the designs.

**Review by greatislander — APPROVED** [review]

**Comment by cherylhjli** [comment]
hey sorry @chosww we have to remove "You have the option to register one or more of the following accounts:” in the registration flow. I'm sorry I didn't catch this earlier - it's pretty misleading UX wise, because users can only register for one of the accounts at a time.

**Pull request merged**